Composer/packages/client/src/recoilModel/selectors/dialogImports.ts

```diff
@@ -37,8 +37,9 @@ export const getLanguageFileImports = <T extends LgFile | LuFile | QnAFile>(
     }
     const currentImports = file.imports.map((item) => {
       const importedFile = getFile(getBaseName(item.id));
+      const displayName = item.id.substring(0, item.id.indexOf('.'));
       return {
-        displayName: item.description,
+        displayName,
```

Comment:
You can keep this mostly as it is and just write `displayName: item.id.substring(0, item.id.indexOf('.'))` as a field on the object (replacing the `item.description` part).